Source/LightboxController.swift

```diff
@@ -321,17 +255,57 @@ extension LightboxController: PageViewDelegate {
 
   func pageVewDidZoom(pageView: PageView) {
     let hidden = pageView.zoomScale != 1.0
+    let duration = hidden ? 0.0 : 1.0
```

Comment:
Could you use `hidden` wrapped in a `CGFloat` here? I'm curious if that would work, but I guess that would look a bit wonky if it would return something that is optional and you need to unwrap it. I dunno, just a thought :)